crates/chia-consensus/src/consensus_constants.rs

```diff
@@ -132,3 +132,71 @@ pub struct ConsensusConstants {
     /// The 32 plot filter adjustment height.
     plot_filter_32_height: u32,
 }
+
+pub const TEST_CONSTANTS: ConsensusConstants = ConsensusConstants {
+    slot_blocks_target: 32,
+    min_blocks_per_challenge_block: 16,
+    max_sub_slot_blocks: 128,
+    num_sps_sub_slot: 64,
+    sub_slot_iters_starting: u64::pow(2, 27),
+
+    difficulty_constant_factor: u128::pow(2, 67),
```

Comment:
is this preferable to `1_u128 << 67`?

Reply:
In the python DEFAULT_CONSTANTS we used `uint128(2**67)` so I kept it equivalent to that